analyzers/src/SonarAnalyzer.Common/Rules/UseUnixEpochBase.cs

```diff
@@ -33,7 +33,8 @@ public abstract class UseUnixEpochBase<TSyntaxKind, TLiteralExpression, TMemberA
     where TLiteralExpression : SyntaxNode
     where TMemberAccessExpression : SyntaxNode
 {
-    private const int EpochYear = 1970;
+    private const long EpochTicks = 0x89F7FF5F7B58000;
```

Comment:
Why did you change this to hexa? In my opinion it is more understandable in decimal. We use that value more often than hexa numbers.

Reply:
I think we should go with decimal with underscores: 621_355_968_000_000_000